lib/collision.js

```diff
@@ -73,30 +72,28 @@ function frogAndCarCollision(item, frog, self){
 
 Collision.prototype.frogCollision = function(frog){
   var self = this;
-  frogs.forEach(function(item, i){
+  frogs.forEach(function(item){
   frogAndCarCollision(item, frog, self);
   });
 };
 
 Collision.prototype.leftCarCollision = function(frog){
   var self = this;
-  leftCar.forEach(function(item, i){
+  leftCar.forEach(function(item){
   frogAndCarCollision(item, frog, self);
   });
 };
 
 Collision.prototype.rightCarCollision = function(frog) {
   var self = this;
-  rightCar.forEach(function(item, i){
+  rightCar.forEach(function(item){
```

Comment:
tried to link these up using this.rightCar during lunch but couldnt make it work